shipyard-controller/common/filereader_test.go

```diff
@@ -0,0 +1,105 @@
+package common
+
+import (
+	"testing"
+	"testing/fstest"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_FileReader(t *testing.T) {
+	denyListTestFileName := "test-file"
+	tests := []struct {
+		provider FileReader
```

Comment:
could you please provide a `name string`? This way the test matrix can have a title for each entry which makes it easier to understand the purpose of the entry